src/docs/design/project-identity.mdx

```diff
@@ -0,0 +1,153 @@
+import { Divider, ImageGrid, MaterialDesignDescription } from '../../app/components';
+import PxblueSmallAltIcon from '@pxblue/icons-mui/PxblueSmallAlt';
+import ConsumerIndustrial from '../../app/assets/design/project-identity/consumer-industrial.png';
+import StyleChoice from '../../app/assets/design/project-identity/style-choice.png';
+import IllustrationConsumer from '../../app/assets/design/project-identity/illustration-consumer.png';
+import IllustrationIndustrial from '../../app/assets/design/project-identity/illustration-industrial.png';
+import ImageAppBar from '../../app/assets/design/project-identity/image-app-bar.png';
+import ImageDrawerRegular from '../../app/assets/design/project-identity/image-drawer-regular.png';
+import ImageDrawerProfile from '../../app/assets/design/project-identity/image-drawer-profile.png';
+import ImageScorecardHeader from '../../app/assets/design/project-identity/image-scorecard-header.png';
+import ImageLogin from '../../app/assets/design/project-identity/image-login.png';
+import ImageSuccess from '../../app/assets/design/project-identity/image-success.png';
+import VariationButton from '../../app/assets/design/project-identity/variation-button.png';
+import VariationTextField from '../../app/assets/design/project-identity/variation-text-field.png';
+import VerbiageNeutral from '../../app/assets/design/project-identity/verbiage-neutral.png';
+import VerbiageExcitement from '../../app/assets/design/project-identity/verbiage-excitement.png';
+import ListColorCorrect from '../../app/assets/design/project-identity/list-color-correct.png';
+import ListColorIncorrect from '../../app/assets/design/project-identity/list-color-incorrect.png';
+
+<!-- KEYWORDS: design tutorial branding brand identity flexible flex -->
+
+# Project Identity
+
+PX Blue provides a foundation for designing and developing applications in a way that makes them feel like they are a part of the same brand. However, the purpose of PX Blue is not to make every application look the same. There is a balance that you must strike between maintaining consistency between applications and establishing a unique brand for each product. PX Blue provides some of the basic elements and patterns to help you build consistency, leaving you more time to focus on what makes your product unique.
+
+<MaterialDesignDescription
+    title={'Design Patterns'}
+    url={'/patterns'}
+    description={`Follow PX Blue's design pattern guidelines.`}
+    avatar={<PxblueSmallAltIcon style={{ width: 48, height: 48 }} />}
+    minCardHeight={'unset'}
+/>
+
+## PX Blue is Flexible
+
+PX Blue is not prescriptive — just as we build on top of Material Design principles, you too can build on top of PX Blue principles to make your products better for your target audience. There are many gray areas and gaps in PX Blue where you are free to make changes and tweaks.
+
+<ImageGrid
+    regularWidth
+    images={[ConsumerIndustrial]}
```

Comment:
The left image does not really look pxblue, IMO. I don't know if we want to stretch it that far for the example, but I'll leave it up to you.

Reply:
It assimilates the consumer-facing project I am recently working on. You may find this image to be very "eric".

I am trying to tone the extended app bar gradient a bit further to include more pxblue blue in the gradient spectrum.

Reply:
I am not surprised to hear that it's an Eric.